test/distrib/bazel/cpp/.bazelrc

```diff
@@ -1,4 +1,15 @@
 # bazelrc file
+common --enable_platform_specific_config
```

Comment:
I'm surprised that this wasn't necessary for the Python Bazel distribtests. Any idea why? I guess `-std=c++14` might just be the default? But then I wouldn't expect the C++ Bazel distribtest to need this bazelrc update.

Reply:
Python bazel distribtests only run on linux. The garden-variety distribtests hide all output unless there's an error, but luckily we've had an error recently, and the logs contain Megabytes of `cl : Command line warning D9002 : ignoring unknown option '-std=c++14'`.

Reply:
I just noticed which file you commented on. Good question, this is mostly unnecessary at the moment, I'm going to revert this file.